functions/src/cpu/parseCPUPage.ts

```diff
@@ -22,7 +23,7 @@ const parseCPUPage = async (
   const specsTable = await getParsingElement('#help_table', page);
 
   const rawSpecsTable = await page.evaluate(async (node) => {
-    async function getNodeTreeText(inputNode: any): Promise<any> {
+    async function getNodeTreeText(inputNode: Element): Promise<string | null> {
       if (inputNode && inputNode.hasChildNodes()) {
         return node.innerText;
```

Comment:
maybe you should reuse this function? import form common

Reply:
declaring functions outside puppeteer's page.evaluate isn't worth a hustle
https://stackoverflow.com/questions/47304665/how-to-pass-a-function-in-puppeteers-evaluate-method